### DISCUSSION TITLE
Upgrading from 2.32.2 to 3.1.2 causes compile errors

### DISCUSSION THREAD

**Discussion by AndrewTraub** [gatsbyjs/gatsby#30710] [Help]
I upgraded my package.json->gatsby from ^2.32.2 to ^3.12 and now gatsby develop crashes with this error:

failed Building development bundle - 59.350s
ERROR in ./src/assets/css/style.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
URIError: URI malformed

**Comment by LekoArts** [comment]
You should check your `src/assets/css/style.css` file then to see what the culprit is.
